OVERVIEW.md

```diff
@@ -0,0 +1,66 @@
+# Overview #
+
+Here it is described a brief overview of xdist's internal architecture.
+
+
+`xdist` works by spawning one or more **worker nodes**, which are controlled
```

Comment:
So using just the terminology you propose:

> `xdist` works by spawning one or more **workers**, which are controlled by the **master**. Each **worker** is responsible for performing a full test collection and afterwards running tests as dictated by the **master**.

Does that sound good? If all agree I will apply the change to the rest of the document.